spring-integration-core/src/test/java/org/springframework/integration/scattergather/config/ScatterGatherParserTests.java

```diff
@@ -17,11 +17,7 @@
 package org.springframework.integration.scattergather.config;
 
 import static org.hamcrest.Matchers.instanceOf;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
No asterisk imports, please.

I'll fix on merge tomorrow.

Thank you for the update!

Reply:
IntelliJ got greedy and did this behind the scenes. Updated now 👍